app-misc/vit/vit-2.2.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{9..11} )
+
+DISTUTILS_SINGLE_IMPL=yes
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="VIT is a lightweight, fast, curses-based front end to Taskwarrior"
+HOMEPAGE="
+	https://github.com/vit-project/vit
+	https://pypi.org/project/vit/
+"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
```

Comment:
Are they really DEPEND? For build-time deps in Python, we use `BDEPEND` but generally you don't need deps at build time (unless build system is doing something silly).